lib/galaxy/webapps/galaxy/api/cloud.py

```diff
@@ -132,7 +133,7 @@ def get(self, trans, payload, **kwargs):
         rtv = []
         for dataset in datasets:
             rtv.append(self.datasets_serializer.serialize_to_view(dataset, view='summary'))
-        return rtv
+        return json.dumps(rtv)
```

Comment:
@VJalili Can you revert and/or rebase over this change as well, since @web.expose_api should be handling this? (if it isn't, we should look into that)

Reply:
Sure, I created a different PR: https://github.com/galaxyproject/galaxy/pull/7592